### DISCUSSION TITLE
Bookmarking post without login

### DISCUSSION THREAD

**Discussion by Thewisem** [Docile-Alligator/Infinity-For-Reddit#949] [Ideas]
Before we could anonymously bookmark without login. But it was removed in new update ans it is an important feature. So why was it removed?

**Comment by Docile-Alligator** [comment]
I didn't remove this feature. What version are you using?

**Comment by Thewisem** [comment]
The latest version on fdroid

https://user-images.githubusercontent.com/74511929/120031709-d5d5f000-c001-11eb-84eb-0a0f1cff7f43.mp4

**Comment by Thewisem** [comment]
Is there a fix on this? Even the new update has it.

**Comment by ghost** [comment]
Hi
I can confirm that local (anonymous) account bookmarks don't seem to be allowed (popup says to connect first) in 5.1.5 from F-droid
This also seems to be referenced in #542 #484 #287. 
Subscriptions do work.

Thanks for the great app !

**Comment by Docile-Alligator** [comment]
Hi, bookmarking is only available in logged-in state. Only subscriptions and multireddits are allowed in anonymous mode.

**Reply by ghost** [reply]
Can anonymous bookmarking be added again?

**Comment by remigermain** [comment]
Hi, it is possible to put back this feature ? Its would be nice , thx

**Comment by ghost** [comment]
I did not know this feature was supposedly removed until recently, I hope there is a way this can be added back soon 🙏
